src/ui/pages/zcl_abapgit_gui_page_sett_remo.clas.abap

```diff
@@ -969,7 +950,9 @@ CLASS zcl_abapgit_gui_page_sett_remo IMPLEMENTATION.
           mo_repo->refresh( ).
         ENDIF.
 
-        rs_handled-state = zcl_abapgit_gui=>c_event_state-go_back_to_bookmark.
+        rs_handled-state = zcl_abapgit_html_form_utils=>create( mo_form )->exit(
+          io_form_data = mo_form_data
+          io_check_changes_versus = initialize_form_data( ) ).
```

Comment:
Switching tab and "back" will trigger the popup because head-type is different, but we can live with that.

Reply:
Yes, also entering values manually and exit does not trigger the change check. It's an issue of different context. Also it was so before I believe. Let's keep it as it is for now.